### PR TITLE
Import expression profile as pkml

### PR DESCRIPTION
**Pull request by georgeDaskalakis**

### CODE REVIEW

**Comment by Yuri05** [comment]
sbml or pkml? :)

**Comment by rwmcintosh** [comment]
> sbml or pkml? :)

pkml

File: src/MoBi.Assets/ToolTips.cs [hunk 1]
```diff
@@ -22,6 +22,11 @@ public static class BuildingBlockMolecule
          public static readonly string AddPKSimMolecule = "Add a PK-Sim Molecule";
       }
 
+      public static class BuildingBlockExpressionProfile
+      {
+         public static readonly string LoadExpressionProfile = $"Load Expression Profile from Expression Profile Building Block file (*{Constants.Filter.PKML_FILE_FILTER})";
```
Comment (msevestre) [inline src/MoBi.Assets/ToolTips.cs:27]: What is this tooltip? WE do not have everything like this

**Review by msevestre — COMMENTED** [review]
Please create an issue to remember to undo the changes in appveyor file

File: src/MoBi.Assets/ToolTips.cs (hunk 1 above)
Comment (msevestre) [reply src/MoBi.Assets/ToolTips.cs:27]: This tooltip look weird with the * etc. Not sure why we have this (or do we have this for other building blocks?)(

**Pull request merged**